src/app/components/Header/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { AppBar, Typography } from '@mui/material';
+import React from 'react';
+import './index.css';
+
+type Props = {
+  children: string;
+};
+
+export default (props: Props): JSX.Element => {
```

Comment:
Do you actually export something here? I would expect something like `export default function Index`.
I assume this has something to do with Material UI?

Reply:
Yes, I do :D it doesn't have anything to do with MUI, it's just an anonymous function. You don't really have to name them.
Thank you for the review! :)